src/components/Pagination/_pagination.scss

```diff
@@ -0,0 +1,64 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-pagination {
+  display: flex;
+  flex-direction: row;
+}
+
+.#{$px} {
+  &--inline-pagination__wrapper {
+    select {
+      appearance: none;
+      border: none;
+      font-size: $body-size2;
+      font-weight: 600;
+      line-height: 2.5rem;
+      margin-bottom: 0;
+      outline: none;
+      text-align: center;
+      text-align-last: center;
+      text-decoration: underline;
+      text-underline-offset: 10px;
```

Comment:
whoa I didn't know about this!  This would have made my `<Link` components much easier to implement!